Library/Formula/sourcegraph.rb

```diff
@@ -0,0 +1,14 @@
+class Sourcegraph < Formula
+  desc "Smarter code host powering top-tier development teams"
+  homepage "https://www.sourcegraph.com"
+  url "https://sourcegraph.com/.download/0.11.2/darwin-amd64/src.gz"
+  sha256 "452d2c1710efeb3c42160280c14e261c14c2de903f49cacd0d2e4bb426c48b8a"
+
+  def install
+    bin.install "src"
+  end
+
+  test do
+    `system "#{bin}/src" "help"`
```

Comment:
This look wrong. I think you meant this to be `system "#{bin}/src", "help"` (without the backticks). Also, can this test be modified to do something more substantial than just printing the help text? Thanks!